include/deal.II/base/tensor.h

```diff
@@ -845,15 +845,15 @@ class Tensor
    * Return an unrolled index in the range $[0,\text{dim}^{\text{rank}}-1]$
    * for the element of the tensor indexed by the argument to the function.
    */
-  static constexpr unsigned int
+  DEAL_II_HOST_DEVICE static constexpr unsigned int
```

Comment:
Is it coincidence that you put `DEAL_II_HOST_DEVICE` as the first argument before `constexpr` here, whereas it comes after `constexpr` in the functions above?

Reply:
yes